src/app/FakeLib/WiXHelper.fs

```diff
@@ -96,13 +100,23 @@ type WiXServiceControl =
         Id : string
         Name: string
         Remove : InstallUninstall
-        Start : InstallUninstall
+        Start : InstallUninstall option
```

Comment:
Yes. But it came just to my mind to extend the InstallUninstall type, like "Never" or "None", instead of option, to accomplish the same task.